### ISSUE TITLE
uv.lib(util.obj) : error LNK2001: unresolved external symbol _GetSystemTimePreciseAsFileTime

### ISSUE THREAD

**Issue by AntonOks** [MoarVM/MoarVM#1789]
- Win11 Pro, Version 23H2
- Microsoft Visual Studio Community 2022 (64-bit) - Current, Version 17.8.6
- C:\Temp\Git\MoarVM>nmake version
  Microsoft (R) Program Maintenance Utility Version 14.38.33135.0
  Copyright (C) Microsoft Corporation.  All rights reserved.

- C:\Temp\Git\MoarVM>cl version
  Microsoft (R) C/C++ Optimizing Compiler Version 19.38.33135 for x86
  Copyright (C) Microsoft Corporation.  All rights reserved.

**Comment by AntonOks** [comment]
@lizmat @MasterDuke17 @jdv @niner 

Any comments on this error?

**Comment by AntonOks** [comment]
Is this here also same as https://github.com/MoarVM/MoarVM/pull/1780#issuecomment-1836909533 ? :|

**Comment by niner** [comment]
This commit https://github.com/MoarVM/MoarVM/commit/428b821d19618f286bacb654bb11fb200c0629f5 was supposed to fix this error

When have you run Configure.pl the last time? Can you please try a completely fresh install?

**Comment by AntonOks** [comment]
> When have you run Configure.pl the last time?

Exactly the day, as I opened this issue.... so ~2 weeks ago.

> Can you please try a completely fresh install?

What exactly do you mean with "fresh install"?
2 weeks ago, I've started with a "git clone..." on my Windows11 (x64 arch, not ARM!) and ran into this error.

**Comment by AntonOks** [comment]
1.
```
C:\Temp\Git\MoarVM>perl Configure.pl
Welcome to MoarVM!

Updating submodules .................................... OK
Configuring native build environment ...................
    trying to compile a simple C program ............... YES
    probing stdatomic .................................. NO
did not find libzstd; will not use heap snapshot format version 3
Defaulting to mimalloc because you are using MSVC
OK

    auto-detecting x64 toolchain ....................... NO
    probing whether your compiler thinks that it is gcc  NO
    probing how your compiler does static inline ....... static inline
    probing if your pow() handles NaN and Inf corner cases  YES
    probing if your log() returns NaN for negative values  YES
    probing if your log10() returns NaN for negative values  YES
    probing if your asin() returns NaN for negative values  YES
    probing if your acos() returns NaN for negative values  YES
    probing if you have isnan .......................... YES
    probing if you have isinf .......................... YES
    probing if you have signbit ........................ YES
    your CPU can read unaligned values for all of int32 int64 num64
    probing the size of pointers ....................... 4
JIT isn't supported on platforms with 4 byte pointers.
    probing C type support for booleans ................ _Bool
    probing computed goto support ...................... NO
    probing pthread_yield support ...................... NO
    probing pthread_setname_np support (optional) ...... NO
    probing existance of optional malloc_trim() ........ NO
    probing support of rdtscp intrinsic ................ YES

        make: nmake
     compile: cl /nologo /MT /std:c17 /Ox /GL /DNDEBUG  /DWIN32 /DAO_ASSUME_WINDOWS98 -DMVM_HEAPSNAPSHOT_FORMAT=2 -D_GNU_SOURCE
    includes:   /I3rdparty/libuv/include /I3rdparty/libuv/src /I3rdparty/libatomicops/src /I3rdparty/libtommath /I3rdparty/dyncall/dynload /I3rdparty/dyncall/dyncall /I3rdparty/dyncall/dyncallback /I3rdparty/mimalloc/include /I3rdparty/mimalloc/src
        link: link /nologo /LTCG
        libs:  shell32.lib ws2_32.lib mswsock.lib rpcrt4.lib advapi32.lib psapi.lib iphlpapi.lib userenv.lib user32.lib bcrypt.lib dbghelp.lib ole32.lib

  byte order: little endian

Checking perl5 modules ................................. OK
Configuring 3rdparty libs .............................. OK

  3rdparty: 3rdparty/cmp/cmp.lib
            3rdparty/dyncall/dyncall/libdyncall_s.lib
            3rdparty/dyncall/dyncallback/libdyncallback_s.lib
            3rdparty/dyncall/dynload/libdynload_s.lib
            3rdparty/libtommath/tommath.lib
            3rdparty/libuv/uv.lib
            3rdparty/sha1/sha1.lib

Generating src/gen/config.c ............................ OK
Generating src/gen/config.h ............................ OK
Generating Makefile .................................... OK
Generating tools/check.mk .............................. OK
Generating build/mk-moar-pc.pl ......................... OK

Configuration SUCCESS.

Type 'nmake' to build and 'nmake help' to see a list of
available make targets.
```

2.
```
C:\Temp\Git\MoarVM>nmake

Microsoft (R) Program Maintenance Utility Version 14.39.33519.0
Copyright (C) Microsoft Corporation.  All rights reserved.

compiling src\main.obj
main.c
compiling src\core\callsite.obj
callsite.c
compiling src\core\args.obj
args.c
compiling src\core\exceptions.obj
exceptions.c
compiling src\core\interp.obj
interp.c
compiling src\core\str_hash_table.obj
str_hash_table.c
compiling src\core\fixkey_hash_table.obj
fixkey_hash_table.c
compiling src\core\index_hash_table.obj
index_hash_table.c
compiling src\core\ptr_hash_table.obj
ptr_hash_table.c
compiling src\core\uni_hash_table.obj
uni_hash_table.c
compiling src\core\threadcontext.obj
threadcontext.c
compiling src\core\compunit.obj
compunit.c
compiling src\core\bytecode.obj
bytecode.c
compiling src\core\frame.obj
frame.c
compiling src\core\callstack.obj
callstack.c
compiling src\core\validation.obj
validation.c
compiling src\core\bytecodedump.obj
bytecodedump.c
compiling src\core\threads.obj
threads.c
compiling src\core\ops.obj
ops.c
compiling src\core\hll.obj
hll.c
compiling src\core\loadbytecode.obj
loadbytecode.c
compiling src\core\coerce.obj
coerce.c
compiling src\core\dll.obj
dll.c
compiling src\core\ext.obj
ext.c
compiling src\core\nativecall.obj
nativecall.c
compiling src\core\nativecall_dyncall.obj
nativecall_dyncall.c
compiling src\core\continuation.obj
continuation.c
compiling src\core\intcache.obj
intcache.c
compiling src\core\regionalloc.obj
regionalloc.c
compiling src\debug\debugserver.obj
debugserver.c
compiling src\gen\config.obj
config.c
compiling src\gc\orchestrate.obj
orchestrate.c
compiling src\gc\allocation.obj
allocation.c
compiling src\gc\worklist.obj
worklist.c
compiling src\gc\roots.obj
roots.c
compiling src\gc\collect.obj
collect.c
compiling src\gc\gen2.obj
gen2.c
compiling src\gc\wb.obj
wb.c
compiling src\gc\objectid.obj
objectid.c
compiling src\gc\finalize.obj
finalize.c
compiling src\gc\debug.obj
debug.c
compiling src\io\io.obj
io.c
compiling src\io\eventloop.obj
eventloop.c
compiling src\io\syncfile.obj
syncfile.c
compiling src\io\syncsocket.obj
syncsocket.c
compiling src\io\fileops.obj
fileops.c
compiling src\io\dirops.obj
dirops.c
compiling src\io\procops.obj
procops.c
compiling src\io\timers.obj
timers.c
compiling src\io\filewatchers.obj
filewatchers.c
compiling src\io\signals.obj
signals.c
compiling src\io\asyncsocket.obj
asyncsocket.c
compiling src\io\asyncsocketudp.obj
asyncsocketudp.c
compiling src\6model\reprs.obj
reprs.c
compiling src\6model\reprconv.obj
reprconv.c
compiling src\6model\containers.obj
containers.c
compiling src\6model\parametric.obj
parametric.c
compiling src\6model\reprs\MVMString.obj
MVMString.c
compiling src\6model\reprs\VMArray.obj
VMArray.c
compiling src\6model\reprs\MVMHash.obj
MVMHash.c
compiling src\6model\reprs\MVMCFunction.obj
MVMCFunction.c
compiling src\6model\reprs\KnowHOWREPR.obj
KnowHOWREPR.c
compiling src\6model\reprs\KnowHOWAttributeREPR.obj
KnowHOWAttributeREPR.c
compiling src\6model\reprs\P6str.obj
P6str.c
compiling src\6model\reprs\P6opaque.obj
P6opaque.c
compiling src\6model\reprs\MVMCode.obj
MVMCode.c
compiling src\6model\reprs\MVMOSHandle.obj
MVMOSHandle.c
compiling src\6model\reprs\MVMCompUnit.obj
MVMCompUnit.c
compiling src\6model\reprs\MVMStaticFrame.obj
MVMStaticFrame.c
compiling src\6model\reprs\P6int.obj
P6int.c
compiling src\6model\reprs\P6num.obj
P6num.c
compiling src\6model\reprs\Uninstantiable.obj
Uninstantiable.c
compiling src\6model\reprs\HashAttrStore.obj
HashAttrStore.c
compiling src\6model\reprs\MVMThread.obj
MVMThread.c
compiling src\6model\reprs\MVMIter.obj
MVMIter.c
compiling src\6model\reprs\MVMContext.obj
MVMContext.c
compiling src\6model\reprs\SCRef.obj
SCRef.c
compiling src\6model\reprs\P6bigint.obj
P6bigint.c
compiling src\6model\reprs\NFA.obj
NFA.c
compiling src\6model\reprs\MVMException.obj
MVMException.c
compiling src\6model\reprs\MVMDLLSym.obj
MVMDLLSym.c
compiling src\6model\reprs\MVMContinuation.obj
MVMContinuation.c
compiling src\6model\reprs\NativeCall.obj
NativeCall.c
compiling src\6model\reprs\CPointer.obj
CPointer.c
compiling src\6model\reprs\CStr.obj
CStr.c
compiling src\6model\reprs\CArray.obj
CArray.c
compiling src\6model\reprs\CStruct.obj
CStruct.c
compiling src\6model\reprs\CUnion.obj
CUnion.c
compiling src\6model\reprs\ReentrantMutex.obj
ReentrantMutex.c
compiling src\6model\reprs\ConditionVariable.obj
ConditionVariable.c
compiling src\6model\reprs\Semaphore.obj
Semaphore.c
compiling src\6model\reprs\ConcBlockingQueue.obj
ConcBlockingQueue.c
compiling src\6model\reprs\MVMAsyncTask.obj
MVMAsyncTask.c
compiling src\6model\reprs\MVMNull.obj
MVMNull.c
compiling src\6model\reprs\CPPStruct.obj
CPPStruct.c
compiling src\6model\reprs\NativeRef.obj
NativeRef.c
compiling src\6model\reprs\MultiDimArray.obj
MultiDimArray.c
compiling src\6model\reprs\Decoder.obj
Decoder.c
compiling src\6model\reprs\MVMSpeshLog.obj
MVMSpeshLog.c
compiling src\6model\reprs\MVMStaticFrameSpesh.obj
MVMStaticFrameSpesh.c
compiling src\6model\reprs\MVMCapture.obj
MVMCapture.c
compiling src\6model\reprs\MVMTracked.obj
MVMTracked.c
compiling src\6model\reprs\MVMStat.obj
MVMStat.c
compiling src\6model\6model.obj
6model.c
compiling src\6model\bootstrap.obj
bootstrap.c
compiling src\6model\sc.obj
sc.c
compiling src\6model\serialization.obj
serialization.c
compiling src\disp\boot.obj
boot.c
compiling src\disp\registry.obj
registry.c
compiling src\disp\inline_cache.obj
inline_cache.c
compiling src\disp\program.obj
program.c
compiling src\disp\syscall.obj
syscall.c
compiling src\disp\resume.obj
resume.c
compiling src\spesh\dump.obj
dump.c
compiling src\spesh\graph.obj
graph.c
compiling src\spesh\codegen.obj
codegen.c
compiling src\spesh\manipulate.obj
manipulate.c
compiling src\spesh\args.obj
args.c
compiling src\spesh\usages.obj
usages.c
compiling src\spesh\facts.obj
facts.c
compiling src\spesh\optimize.obj
optimize.c
compiling src\spesh\dead_bb_elimination.obj
dead_bb_elimination.c
compiling src\spesh\dead_ins_elimination.obj
dead_ins_elimination.c
compiling src\spesh\deopt.obj
deopt.c
compiling src\spesh\log.obj
log.c
compiling src\spesh\threshold.obj
threshold.c
compiling src\spesh\inline.obj
inline.c
compiling src\spesh\osr.obj
osr.c
compiling src\spesh\lookup.obj
lookup.c
compiling src\spesh\iterator.obj
iterator.c
compiling src\spesh\worker.obj
worker.c
compiling src\spesh\debug.obj
debug.c
compiling src\spesh\stats.obj
stats.c
compiling src\spesh\plan.obj
plan.c
compiling src\spesh\arg_guard.obj
arg_guard.c
compiling src\spesh\frame_walker.obj
frame_walker.c
compiling src\spesh\pea.obj
pea.c
compiling src\6model\reprs\MVMSpeshCandidate.obj
MVMSpeshCandidate.c
compiling src\spesh\disp.obj
disp.c
compiling src\strings\decode_stream.obj
decode_stream.c
compiling src\strings\ascii.obj
ascii.c
compiling src\strings\parse_num.obj
parse_num.c
compiling src\strings\utf8.obj
utf8.c
compiling src\strings\utf8_c8.obj
utf8_c8.c
compiling src\strings\nfg.obj
nfg.c
compiling src\strings\ops.obj
ops.c
compiling src\strings\unicode.obj
unicode.c
compiling src\strings\normalize.obj
normalize.c
compiling src\strings\latin1.obj
latin1.c
compiling src\strings\utf16.obj
utf16.c
compiling src\strings\windows1252.obj
windows1252.c
compiling src\strings\shiftjis.obj
shiftjis.c
compiling src\strings\gb2312.obj
gb2312.c
compiling src\strings\gb18030.obj
gb18030.c
compiling src\math\bigintops.obj
bigintops.c
compiling src\profiler\instrument.obj
instrument.c
compiling src\profiler\log.obj
log.c
compiling src\profiler\profile.obj
profile.c
compiling src\profiler\heapsnapshot.obj
heapsnapshot.c
compiling src\profiler\telemeh.obj
telemeh.c
compiling src\profiler\configuration.obj
configuration.c
compiling src\instrument\crossthreadwrite.obj
crossthreadwrite.c
compiling src\instrument\line_coverage.obj
line_coverage.c
compiling src\platform\sys.obj
sys.c
compiling src\platform\random.obj
random.c
src\platform\random.c(133): warning C4113: 'FARPROC' differs in parameter lists from 'CRYPTGENRANDOM'
src\platform\random.c(131): warning C4113: 'FARPROC' differs in parameter lists from 'CRYPTACQUIRECONTEXTA'
compiling src\platform\memmem32.obj
memmem32.c
compiling 3rdparty\freebsd\memmem.obj
memmem.c
compiling 3rdparty\ryu\ryu\d2s.obj
d2s.c
compiling 3rdparty\mimalloc\src\static.obj
static.c
compiling src\platform\malloc_trim.obj
malloc_trim.c
compiling src\moar.obj
moar.c
src\moar.c(115): warning C4113: 'void (__cdecl *)()' differs in parameter lists from 'void (__cdecl *)(void)'
compiling src\platform\win32\mmap.obj
mmap.c
compiling src\platform\win32\time.obj
time.c
compiling src\platform\win32\io.obj
io.c
compiling src\platform\win32\fork.obj
fork.c
compiling src\jit\stub.obj
stub.c
compiling 3rdparty\libuv\src\idna.obj
idna.c
compiling 3rdparty\libuv\src\timer.obj
timer.c
compiling 3rdparty\libuv\src\win\fs-event.obj
fs-event.c
compiling 3rdparty\libuv\src\win\fs.obj
fs.c
compiling 3rdparty\libuv\src\win\pipe.obj
pipe.c
compiling 3rdparty\libuv\src\win\process.obj
process.c
compiling 3rdparty\libuv\src\win\tty.obj
tty.c
compiling 3rdparty\libuv\src\win\util.obj
util.c
linking 3rdparty\libuv\uv.lib
linking moar.dll
   Creating library moar.dll.lib and object moar.dll.exp
uv.lib(util.obj) : error LNK2001: unresolved external symbol _GetSystemTimePreciseAsFileTime
moar.dll : fatal error LNK1120: 1 unresolved externals
NMAKE : fatal error U1077: '@link /out:moar.dll /nologo /LTCG /dll /implib:moar.dll.lib src\core\callsite.obj  src\core\args.obj  src\core\exceptions.obj  src\core\interp.obj  src\core\str_hash_table.obj  src\core\fixkey_hash_table.obj  src\core\index_hash_table.obj  src\core\ptr_hash_table.obj  src\core\uni_hash_table.obj  src\core\threadcontext.obj  src\core\compunit.obj  src\core\bytecode.obj  src\core\frame.obj  src\core\callstack.obj  src\core\validation.obj  src\core\bytecodedump.obj  src\core\threads.obj  src\core\ops.obj  src\core\hll.obj  src\core\loadbytecode.obj  src\core\coerce.obj  src\core\dll.obj  src\core\ext.obj  src\core\nativecall.obj  src\core\nativecall_dyncall.obj  src\core\continuation.obj  src\core\intcache.obj  src\core\regionalloc.obj  src\debug\debugserver.obj  src\gen\config.obj  src\gc\orchestrate.obj  src\gc\allocation.obj  src\gc\worklist.obj  src\gc\roots.obj  src\gc\collect.obj  src\gc\gen2.obj  src\gc\wb.obj  src\gc\objectid.obj  src\gc\finalize.obj  src\gc\debug.obj  src\io\io.obj  src\io\eventloop.obj  src\io\syncfile.obj  src\io\syncsocket.obj  src\io\fileops.obj  src\io\dirops.obj  src\io\procops.obj  src\io\timers.obj  src\io\filewatchers.obj  src\io\signals.obj  src\io\asyncsocket.obj  src\io\asyncsocketudp.obj  src\6model\reprs.obj  src\6model\reprconv.obj  src\6model\containers.obj  src\6model\parametric.obj  src\6model\reprs\MVMString.obj  src\6model\reprs\VMArray.obj  src\6model\reprs\MVMHash.obj  src\6model\reprs\MVMCFunction.obj  src\6model\reprs\KnowHOWREPR.obj  src\6model\reprs\KnowHOWAttributeREPR.obj  src\6model\reprs\P6str.obj  src\6model\reprs\P6opaque.obj  src\6model\reprs\MVMCode.obj  src\6model\reprs\MVMOSHandle.obj  src\6model\reprs\MVMCompUnit.obj  src\6model\reprs\MVMStaticFrame.obj  src\6model\reprs\P6int.obj  src\6model\reprs\P6num.obj  src\6model\reprs\Uninstantiable.obj  src\6model\reprs\HashAttrStore.obj  src\6model\reprs\MVMThread.obj  src\6model\reprs\MVMIter.obj  src\6model\reprs\MVMContext.obj  src\6model\reprs\SCRef.obj  src\6model\reprs\P6bigint.obj  src\6model\reprs\NFA.obj  src\6model\reprs\MVMException.obj src\6model\reprs\MVMDLLSym.obj  src\6model\reprs\MVMContinuation.obj  src\6model\reprs\NativeCall.obj  src\6model\reprs\CPointer.obj  src\6model\reprs\CStr.obj  src\6model\reprs\CArray.obj  src\6model\reprs\CStruct.obj  src\6model\reprs\CUnion.obj  src\6model\reprs\ReentrantMutex.obj  src\6model\reprs\ConditionVariable.obj  src\6model\reprs\Semaphore.obj  src\6model\reprs\ConcBlockingQueue.obj  src\6model\reprs\MVMAsyncTask.obj  src\6model\reprs\MVMNull.obj  src\6model\reprs\CPPStruct.obj  src\6model\reprs\NativeRef.obj  src\6model\reprs\MultiDimArray.obj  src\6model\reprs\Decoder.obj  src\6model\reprs\MVMSpeshLog.obj  src\6model\reprs\MVMStaticFrameSpesh.obj  src\6model\reprs\MVMCapture.obj  src\6model\reprs\MVMTracked.obj  src\6model\reprs\MVMStat.obj  src\6model\6model.obj  src\6model\bootstrap.obj  src\6model\sc.obj  src\6model\serialization.obj  src\disp\boot.obj  src\disp\registry.obj  src\disp\inline_cache.obj  src\disp\program.obj  src\disp\syscall.obj  src\disp\resume.obj  src\spesh\dump.obj  src\spesh\graph.obj  src\spesh\codegen.obj  src\spesh\manipulate.obj  src\spesh\args.obj  src\spesh\usages.obj  src\spesh\facts.obj  src\spesh\optimize.obj  src\spesh\dead_bb_elimination.obj  src\spesh\dead_ins_elimination.obj  src\spesh\deopt.obj  src\spesh\log.obj  src\spesh\threshold.obj  src\spesh\inline.obj  src\spesh\osr.obj  src\spesh\lookup.obj  src\spesh\iterator.obj  src\spesh\worker.obj  src\spesh\debug.obj  src\spesh\stats.obj  src\spesh\plan.obj  src\spesh\arg_guard.obj  src\spesh\frame_walker.obj  src\spesh\pea.obj  src\6model\reprs\MVMSpeshCandidate.obj  src\spesh\disp.obj  src\strings\decode_stream.obj  src\strings\ascii.obj  src\strings\parse_num.obj  src\strings\utf8.obj  src\strings\utf8_c8.obj  src\strings\nfg.obj  src\strings\ops.obj  src\strings\unicode.obj  src\strings\normalize.obj  src\strings\latin1.obj  src\strings\utf16.obj  src\strings\windows1252.obj  src\strings\shiftjis.obj  src\strings\gb2312.obj  src\strings\gb18030.obj  src\math\bigintops.obj  src\profiler\instrument.obj  src\profiler\log.obj  src\profiler\profile.obj  src\profiler\heapsnapshot.obj  src\profiler\telemeh.obj  src\profiler\configuration.obj  src\instrument\crossthreadwrite.obj  src\instrument\line_coverage.obj  src\platform\sys.obj  src\platform\random.obj  src\platform\memmem32.obj  3rdparty\freebsd\memmem.obj  3rdparty\ryu\ryu\d2s.obj  3rdparty\mimalloc\src\static.obj  src\platform\malloc_trim.obj  src\moar.obj  src\platform\win32\mmap.obj  src\platform\win32\time.obj  src\platform\win32\io.obj  src\platform\win32\fork.obj  src\jit\stub.obj 3rdparty\cmp\cmp.lib 3rdparty\dyncall\dyncall\libdyncall_s.lib 3rdparty\dyncall\dyncallback\libdyncallback_s.lib 3rdparty\dyncall\dynload\libdynload_s.lib 3rdparty\sha1\sha1.lib 3rdparty\libtommath\tommath.lib 3rdparty\libuv\uv.lib shell32.lib ws2_32.lib mswsock.lib rpcrt4.lib advapi32.lib psapi.lib iphlpapi.lib userenv.lib user32.lib bcrypt.lib dbghelp.lib ole32.lib' : return code '0x460'
Stop.

C:\Temp\Git\MoarVM>
```

... so, still the same error ...

**Comment by ugexe** [comment]
The issue started with a libuv bump. I suspect we might need to adjust the libuv files included (or their ordering) in our build system.

**Comment by raiph** [comment]
@AntonOks 

> :( Getting this error reported from my Win11 :|

My 2c:

I get that you may associate that with this issue, perhaps your attempts to progress it, but it seems like something that A) should be dealt with independently of this issue for now (even if it turns out to be related), B) possibly a misreport, possibly benign, possibly malign, and thus both possibly no big deal or, conversely, an important and urgent issue, C) something other readers of this issue are ignoring, so something you should discuss elsewhere, preferably in accord with whatever written security policies we have in place for Raku (I recall a mention of a policy, including where to post such things, being updated recently).

**Comment by AntonOks** [comment]
> @AntonOks
> 
> > :( Getting this error reported from my Win11 :|
> 
> My 2c:
> 
> I get that you may associate that with this issue, perhaps your attempts to progress it, but it seems like something that A) should be dealt with independently of this issue for now (even if it turns out to be related), B) possibly a misreport, possibly benign, possibly malign, and thus both possibly no big deal or, conversely, an important and urgent issue, C) something other readers of this issue are ignoring, so something you should discuss elsewhere, preferably in accord with whatever written security policies we have in place for Raku (I recall a mention of a policy, including where to post such things, being updated recently).

???

I just want to share / report that this "software" is not compiling as it is said to be.
And the usual place (for GitHub repos) for such reports is this "issues" place here. right or wrong?

For sure one can post "issues" everywhere in the internet... and maybe Google and friends would reveal those posts for the next one who faces the same issue... but, well, ....

**Comment by raiph** [comment]
@AntonOks

>> My 2c:

> ???

OK, my apologies, I realize I misinterpreted your intent.

(For some reason -- the thought that jumped out as I wrote that was a reaction to the emojis you used -- I incorrectly jumped to the thought you were implicitly seeking some specific follow up. I even recall presuming you were a bit concerned about your own system, and wanting follow up for that reason. But I now realize it was more of an FYI, such that it was up to those who read it to do as they saw fit. So the fact I was concerned about -- it's a bit invisible due to it being written in this issue, which isn't about a security issue, and then ugexe appearing to ignore it and instead focus on the topic of this issue -- is not your problem. That makes sense, and I now understand it, and apologize to you and others for the noise.)

🙏

**Comment by AntonOks** [comment]
> The issue started with a libuv bump. I suspect we might need to adjust the libuv files included (or their ordering) in our build system.

@ugexe 
Don't really understand which "adjustment"would be an option and where to do them, but if I can try something out here on my box... happy to help, pls let me know.

**Comment by AntonOks** [comment]
I guess I've found the problem... it may even be a "works as designed", not sure... 

my issue is related to my `Developer Command Prompt...`, which opens a `x86` environment. I should have notices this already right away because the `cl version`, see my original error posting above, clearly says `... for x86` :(

Recently I was compiling MoarVM a couple of times, based on different `libuv` submodule commits,... and every time the MoarVM compilation "just worked". It took me some time to realize, I was now working with the `Developer PoweShell...`, which opens a `x64` environment... and so the related `cl`... 

So, in short, the issue is the x86 cl... 

Btw., could / should this somehow be checked / captured by the Rakudo / NQP / MoarVM build process, similar to https://github.com/Raku/nqp-configure/blob/1b4ae93d39f6f5151da1b0d303becd26f7c318b7/lib/NQP/Config.pm#L254

**Comment by lizmat** [comment]
@zhuomingliang
